easybuild/easyblocks/t/tkinter.py

```diff
@@ -77,14 +83,11 @@ def install_step(self):
 
         copy([os.path.join(pylibdir, x) for x in tkparts], tmpdir)
 
-        rmtree2(self.installdir)
+        remove_dir(self.installdir)
 
-        mkdir(pylibdir, parents=True)
-        try:
-            shutil.move(os.path.join(tmpdir, tkparts[0]), pylibdir)
-            shutil.move(os.path.join(tmpdir, os.path.basename(tkparts[1])), pylibdir)
-        except (IOError, OSError) as err:
-            raise EasyBuildError("Failed to move Tkinter back to the install directory: %s", err)
+        move_file(os.path.join(tmpdir, tkparts[0]), os.path.join(pylibdir, tkparts[0]))
```

Comment:
Can't move_file take a dir as the second arg? Because if it can you should only need pylibdir there and not the path.join with tkparts[0].

And the same for the one below.

Reply:
Right now, it can't.

The problem is that you can't distinguish between a file or directory if the target doesn't exist yet in case you're moving a file: should the target directory be created, and the file moved into it, or should the parent of the target be created as a directory, and then the file moved to the specified target?

Reply:
Aha, forgot the non-exist dir situation...